### PR TITLE
S3t8 get+display events

### PR DESCRIPTION
**Pull request by RachmannJoubert**
Functions for retireving the events information from the database and displaying it on the vents page.

### CODE REVIEW

File: public/js/getEvents.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+        .then(eventInfo => eventInfo.json())
+        .then(eventInfo => displayEventsHandler(eventInfo.data))
+}
+
+/**
+ * Runs a foreach through each event object and outputs HTML elements with event details
+ *
+ * @param events is an array of objects which contains information about events
+ *
+ * @return a divs of the event name with a button that reveals each event's additional info on each line
```
Comment (FelixLetheren) [inline public/js/getEvents.js]: should this be 'div'? Doesn't make sense to me.

File: public/js/getEvents.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+        },
+    })
+        .then(eventInfo => eventInfo.json())
+        .then(eventInfo => displayEventsHandler(eventInfo.data))
+}
+
+/**
+ * Runs a foreach through each event object and outputs HTML elements with event details
+ *
+ * @param events is an array of objects which contains information about events
```
Comment (FelixLetheren) [inline public/js/getEvents.js]: should be typehinted to array?

Comment (mallowmew) [reply public/js/getEvents.js]: can't type hint in js

File: public/js/getEvents.js [hunk 3]
```diff
@@ -0,0 +1,58 @@
+/**
+ * Gets event information from the API and passes into the displayHandler function
+ *
+ * @return event data
+ */
+async function getEvents() {
```
Comment (mallowmew) [inline public/js/getEvents.js]: Does not need to be async/await

File: public/js/getEvents.js (hunk 1 above)
Comment (mallowmew) [reply public/js/getEvents.js]: It doesn't return anything, it spits out html

File: templates/EventsPage.phtml [hunk 4]
```diff
@@ -41,7 +42,7 @@
         <label for='event-notes'>Notes</label>
         <input id='event-notes' name='event-notes' class="create-events form-control" type="text" placeholder="Notes"
                data-max='255'>
-        <input id="submit-event" type="submit" class="btn btn-success"/>
+        <input id="create-events" type="submit" class="btn btn-success"/>
```
Comment (mallowmew) [inline templates/EventsPage.phtml]: Why did this change?

File: templates/EventsPage.phtml [hunk 5]
```diff
@@ -41,7 +42,7 @@
         <label for='event-notes'>Notes</label>
         <input id='event-notes' name='event-notes' class="create-events form-control" type="text" placeholder="Notes"
                data-max='255'>
-        <input id="submit-event" type="submit" class="btn btn-success"/>
+        <input id="submit-events" type="submit" class="btn btn-success"/>
```
Comment (mallowmew) [inline templates/EventsPage.phtml]: You submit a single event at a time

**Pull request merged**